docs/site/HasOne-relation.md

```diff
@@ -508,14 +570,15 @@ export class SupplierRepository extends DefaultCrudRepository {
       account: {accountManager: 'Odin', supplierId: 1},
     },
     {
-      id: 5,
+      id: 2,
       name: 'Loki',
-      account: {accountManager: 'Frigga', supplierId: 5},
+      account: {accountManager: 'Frigga', supplierId: 2},
```

Comment:
I used `accountManager` instead of `name` because of [hasOne-relation-graph.png](https://github.com/strongloop/loopback-next/blob/master/docs/site/imgs/hasOne-relation-graph.png)

Reply:
I see, I guess line 171 needs to be reverted to `accountManager` and the `supplierId` here should be 5 to match the diagram. ( I totally forgot about those diagrams :P ).